Frontend/ui-common/src/Resource/Constants.purs

```diff
@@ -24,4 +24,7 @@ chatService :: String
 chatService = "in.juspay.mobility.app.ChatService"
 
 locateOnMapLabelMaxWidth :: Int
-locateOnMapLabelMaxWidth = if (os == "IOS") then 140 else 400
\ No newline at end of file
+locateOnMapLabelMaxWidth = if (os == "IOS") then 140 else 400
+
+maxImageUploadInIssueReporting :: Int  
```

Comment:
Is this variable used in issue reporting flow ?

Reply:
yea , i do this because in image upload model maximum images you can upload is 3 but after this you can upload maximum 4 images